cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala

```diff
@@ -0,0 +1,292 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.cypher.docgen
+
+import org.neo4j.cypher.docgen.tooling._
+
+class TransactionsCommandTest extends DocumentingTest {
+
+  override def outputPath = "target/docs/dev/ql/listing"
+
+  override def doc: Document = new DocBuilder {
+    doc("Transaction Commands", "query-transaction-clauses")
+    synopsis("This section explains the `SHOW TRANSACTIONS` and `TERMINATION TRANSACTIONS` commands.")
+    section("SHOW TRANSACTIONS", id="query-listing-transactions") {
+      p("The `SHOW TRANSACTIONS` command is used to display running transactions.")
+      p(
+        """
+          #[NOTE]
+          #====
+          #The command `SHOW TRANSACTIONS` only outputs the default output; for a full output use the optional `YIELD` command.
+          #Full output: `SHOW TRANSACTIONS YIELD *`.
+          #====""".stripMargin('#'))
+      p("This command will produce a table with the following columns:")
+      p(
+        """
+.List transactions output
+[options="header", cols="4,6,2"]
+||===
+|| Column
+|| Description
+|| Type
+
+|m|database
+|a|This is the name of the database the transaction is executing against. label:default-output[]
+|m|STRING
+
+|m|transactionId
+|a|This is the ID of the transaction. label:default-output[]
+|m|STRING
+
+|m|currentQueryId
+|a|This is the ID of the current query executed by transaction. label:default-output[]
+|m|STRING
+
+|m|connectionId
+|a|The ID of the database connection attached to the transaction or blank for embedded connections. label:default-output[]
+|m|STRING
+
+|m|clientAddress
+|a|The client address of the connection issuing the transaction, or blank if this is not available. label:default-output[]
+|m|STRING
+
+|m|username
+|a|This is the username of the user who is executing the transaction. label:default-output[]
+|m|STRING
+
+|m|currentQuery
+|a|The query text of the query of the current query executed by the transaction. label:default-output[]
```

Comment:
I am not sure I understand it but perhaps you want to say something like:
```suggestion
|a|The query text of the query currently executing in this transaction. label:default-output[]
```